src/app/in-memory-data.service.ts

```diff
@@ -1,9 +1,10 @@
 import { InMemoryDbService } from 'angular-in-memory-web-api';
 import { projectBuilds } from './mock/project-builds';
 import { projects } from './mock/projects';
+import { builds } from './mock/builds';
 
 export class InMemoryDataService implements InMemoryDbService {
   createDb() {
-    return { projectBuilds, projects };
+    return { projectBuilds, projects, builds };
```

Comment:
we ditched the in memory data service for a different mock data service Radu put together. you'll want to rebase sooner or later before it gets too painful.